rasa/nlu/training_data/formats/readerwriter.py

```diff
@@ -10,8 +10,12 @@
 
 
 class TrainingDataReader:
+
+    filename = ""
```

Comment:
We should move this into an `__init__` so that it's more consistent with other classes.